libbeat/docs/dashboards.asciidoc

```diff
@@ -3,43 +3,30 @@
 
 Kibana has a large set of visualization types that you can combine to create
 the perfect dashboards for your needs. But this flexibility can be a bit
-overwhelming at the beginning, so we have created a couple of
-https://github.com/elastic/beats-dashboards[Sample Dashboards] to get you
+overwhelming at the beginning, so we have created some sample dashboards to get you
 started and to demonstrate what is possible based on the Beats data.
 
 NOTE: The Beats dashboards require Kibana 4 or later.
 
-To load the sample dashboards, you run a load script. The load script
-uploads the example dashboards, visualizations, and searches
-that you can use. The load command also creates index patterns for each Beat:
+Each Beat comes packaged with a script that you can use to import the example dashboards,
+visualizations, and searches for that Beat. The script also creates index patterns, such as 
+`packetbeat-*` for Packetbeat, `topbeat-*` for Topbeat, and so on. 
 
-   - `packetbeat-*`
-   - `topbeat-*`
-   - `filebeat-*`
-   - `winlogbeat-*`
-   
-Use the following commands to run the script:
+Use the following commands to run the script from the directory where you installed the Beat:
 
 *deb, rpm, and mac:*
 
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
-curl -L -O http://download.elastic.co/beats/dashboards/beats-dashboards-{Dashboards-version}.zip
-unzip beats-dashboards-{Dashboards-version}.zip
-cd beats-dashboards-{Dashboards-version}/
-./load.sh
+cd kibana/
```

Comment:
The exact location of this `kibana` folder depends on the install type. It's in the "home path", so for deb and rpm it should be:

```
cd /usr/share/{{beatname_lc}}/kibana/
```

For everything else, `cd kibana/` is correct, as it will be in the extracted path.